### PR TITLE
chore: add README.md for opentelemetry-cloud-trace-propagator

### PR DESCRIPTION
**Pull request by shivkanya9146**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue/feature](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Split from #116

### CODE REVIEW

File: packages/opentelemetry-cloud-trace-propagator/README.md [hunk 1]
```diff
@@ -1,11 +1,44 @@
-# `@google-cloud/opentelemetry-cloud-trace-propagator`
+# OpenTelemetry Google Cloud Trace Propagator
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
 
-> TODO: description
+OpenTelemetry Google Cloud Trace Propagator allow other services to create spans with the right context.
+
+Format:
+TRACE_ID/SPAN_ID;o=TRACE_TRUE
```
Comment (mayurkale22) [inline packages/opentelemetry-cloud-trace-propagator/README.md:8]:
```suggestion
`TRACE_ID/SPAN_ID;o=TRACE_TRUE`
```

File: packages/opentelemetry-cloud-trace-propagator/README.md [hunk 2]
```diff
@@ -1,11 +1,44 @@
...
+  propagator: new CloudPropagator()
+});
 ```
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
+- Learn more about Google Cloud Trace at https://cloud.google.com/trace
+
+
```
Comment (mayurkale22) [inline packages/opentelemetry-cloud-trace-propagator/README.md:40]: minor: remove extra line.

File: packages/opentelemetry-cloud-trace-propagator/README.md (hunk 1 above)
Comment (shivkanya9146) [reply packages/opentelemetry-cloud-trace-propagator/README.md]: done.

File: packages/opentelemetry-cloud-trace-propagator/README.md (hunk 2 above)
Comment (shivkanya9146) [reply packages/opentelemetry-cloud-trace-propagator/README.md]: done.

**Pull request merged**